lib/charms/data_platform_libs/v0/data_interfaces.py

```diff
@@ -334,6 +334,10 @@ def __init__(self, *args):
         self.framework.observe(
             self.karapace.on.subject_allowed, self._on_karapace_subject_allowed
         )
+        self.framework.observe(
+            self.karapace.on.karapace_entity_created, self._on_karapace_entity_created
```

Comment:
Careful! Following the naming of other Data Platform products, this should be:

```suggestion
            self.karapace.on.subject_entity_created, self._on_subject_entity_created
```